helm-charts/konk-service/templates/configmap.yaml

```diff
@@ -25,18 +22,32 @@ data:
     echo 'Done' > /tmp/healthy
 
   api-service.yaml.in: |
+    apiVersion: v1
+    kind: Namespace
+    metadata:
+      name: {{` {{ NAMESPACE }} `}}
+    ---
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: {{` {{ SERVICENAME }} `}}
+      namespace: {{` {{ NAMESPACE }} `}}
+    spec:
+      type: ExternalName
+      externalName: {{` {{ SERVICENAME }} `}}.{{` {{ NAMESPACE }} `}}
+    ---
     apiVersion: apiregistration.k8s.io/v1
     kind: APIService
     metadata:
-      name: v1alpha1.{{` {{ SERVICENAME }} `}}.infoblox.com
+      name: {{ .Values.version }}.{{ .Values.group }}
```

Comment:
This worked? 😮 Interesting.

Reply:
This part of the template is evaluated by the helm-operator at the time it deploys the chart from the CR. So as long as the value is set in the CR, it will get populated here. It wouldn't work for values generated at runtime like the cert contents.

Reply:
Oh, this is just the name, nvm.